bitnami/rabbitmq/templates/networkpolicy.yaml

```
@@ -59,6 +59,9 @@ spec:
         - port: {{ .Values.containerPorts.amqpTls }}
         - port: {{ .Values.containerPorts.dist }}
         - port: {{ .Values.containerPorts.manager }}
+        {{ - range .Values.extraContainerPorts }}
+        - port: {{ . }}
+        {{ - end }}
```

This template is invalid, it should use `{{-` not `{{ -`
```suggestion
        {{- range .Values.extraContainerPorts }}
        - port: {{ . }}
        {{- end }}
```

---

OK thanks, sorry about that. I have pushed the updated template formatting.